db/schema.rb

```diff
@@ -10,64 +10,32 @@
 #
```

Comment:
Coudl you unstage the schem file from your commit? I'm just weary that it might break something if you did not make changess and its detecting a conflict

Reply:
I tried but it is not working. All the commits have three files,